### DISCUSSION TITLE
What are some essential SoilC and GHG flux data sources that we should use?

### DISCUSSION THREAD

**Discussion by dlebauer** [ccmmf/organization#14] [Q&A]
Please list: 
- variables measured including units
- temporal + spatial resolution
- temporal + spatial domain
- other relevant information and questions for discussion
- reference(s)
- links to data source(s)

**Comment by dlebauer** [comment]
Gridded National Inventory of U.S. Methane Emissions

[See map](https://earth.gov/ghgcenter/exploration?search=&datasets=%5B%7B%22id%22%3A%22rice-cultivation-l%22%2C%22settings%22%3A%7B%22isVisible%22%3Atrue%2C%22opacity%22%3A100%2C%22analysisMetrics%22%3A%5B%7B%22id%22%3A%22mean%22%2C%22label%22%3A%22Average%22%2C%22chartLabel%22%3A%22Average%22%2C%22themeColor%22%3A%22infographicB%22%7D%5D%2C%22colorMap%22%3A%22epa-ghgi-ch4%22%2C%22reScale%22%3A%7B%22min%22%3A0%2C%22max%22%3A20%7D%2C%22scale%22%3A%7B%22min%22%3A0%2C%22max%22%3A20%7D%7D%7D%5D&taxonomy=%7B%7D&date=2020-11-20T07%3A00%3A00.000Z):
- focus on Rice, ignore other variables
- 0.1 degree grid
- Download raster for each year and compute Rice CH4 inventories to compare w/ MAGiC inventories
- **Note** this is another model product, so should not be treated as data for validation.

Methane
- CH4 flux (Mg CH₄/km²/yr)
- Temporal Resolution and domain: Annual (2012-2020)
- Spatial Resolution and Domain: CONUS, 0.1 degree grid 
- https://earth.gov/ghgcenter/data-catalog/epa-ch4emission-yeargrid-v2express
- Maasakkers et al 2016 Gridded National Inventory of U.S. Methane Emissions. EST. https://pubs.acs.org/doi/epdf/10.1021/acs.est.6b02878 

> Annual state-level emissions from rice cultivation are obtained from EPA and allocated to counties using acreage harvested from the USDA Census. [(14)](javascript:void(0);) Emissions for each county are allocated to the 0.1° × 0.1° grid based on crop maps with 30 m resolution from the USDA Cropland Data Layer product. Annual emissions are then distributed over individual months using normalized mean 2001–2010 heterotrophic respiration rates from the 1° × 1° monthly Carbon Data-Model Framework (CARDAMOM) terrestrial C cycle analysis. 
Emissions from field burning of agricultural residues of five individual crops (corn, rice, soybeans, sugar cane, and wheat) are allocated to a 2003–2007 monthly climatology of agricultural fires.

**Reply by dlebauer** [reply]
Moved to https://github.com/ccmmf/organization/issues/225

**Comment by dlebauer** [comment]
Here is an inventory of Ameriflux sites with GHG flux measurements: CH4 (26 sites) and N2O (14 sites) 

- CH4 and N2O flux measurements
- site based, sub-daily
- this only covers US sites; varying temporal domain
- compiled from the search interface https://ameriflux.lbl.gov/sites/site-search/
  - NB: site names may have typos - I was just counting the sites while writing the proposal and noted these down, and transcribed them here so they don't get lost 

<details>

| Site   | CH4 | N2O | Crop                                                             |
|--------|-----|-----|------------------------------------------------------------------|
| US-Bi1 | X   | X   | corn                                                             |
| US-Bi2 | X   | X   | soy, winter wheat                                                |
| US-CRT | X   |     | annual and perennial croplands - alfalfa, soybean, wheat, cattle |
| US-HRA | X   |     | rice                                                             |
| US-HRC | X   |     | rice                                                             |
| US-TWC | X   |     | alfalfa                                                          |
| US-Twt | X   |     | rice                                                             |
| US-XSL | X   |     | mixed                                                            |
| US-Ro6 |     | X   | sorn, soy, clover                                                |
| US-RO5 |     | X   | corn, soy                                                        |
| US-RGW | X   | X   | rice                                                             |
| US-RGO | X   | X   | rice                                                             |
| US-RGF | X   | X   | corn, wheat with manure application                              |
| US-RGB | X   | X   | rice                                                             |
| US-RGA | X   | X   | bioenergy crops                                                  |
| US-DSB | X   | X   | rice                                                             |
| US-ARM | X   | X   | wheat, corn, soy, alfalfa                                        |
| US-ER1 | X   | X   | not stated - "ag land"                                           |
| US-ER2 |     | X   | corn                                                             |
| US-BD4 | X   |     | rice                                                             |
| US-BD5 | X   |     | rice                                                             |
| US-BD6 | X   |     | rice                                                             |
| US-DS3 | X   | X   | rice                                                             |
| US-BdC | X   |     | rice, soy                                                        |
| US-BdA | X   |     | rice, soy                                                        |
| US-OF1 | X   |     | rice, soy                                                        |
| US-OF2 | X   |     | rice, soy                                                        |
| US-RGG | X   |     | rice                                                             |
| US-AWD | X   |     | rice                                                             |

</details>

[Ameriflux_N2O_CH4_Sites.csv](https://github.com/user-attachments/files/16732518/Ameriflux_N2O_CH4_Sites.csv)

**Comment by dlebauer** [comment]
Inventory

CALGEM https://ww2.arb.ca.gov/ghg-inventory-data

- N2O, CH4 (along with other GHGs)
- Annual, state level, 2000-2021) 
  - broken out by source (incl. mineral fertilizer, org fertilizer, manure, rice fields)

Can provide a good point of comparison of our state level estimates.

**Comment by dlebauer** [comment]
Satellite based measurements

Overview:
[Jacob, Daniel J., et al.](https://acp.copernicus.org/articles/22/9617/2022/acp-22-9617-2022-discussion.html) "Quantifying methane emissions from the global scale down to point sources using satellite observations of atmospheric methane." Atmospheric Chemistry and Physics 22.14 (2022): 9617-9646. 

Of these, [MethaneSat](https://www.edf.org/methanesat?gad_source=1) is the only one specifically identified as detecting agriculture.

Unclear whether this will be useful for ag applications. Data not yet available.

- CH4 flux
- resolution
  - spatial: 1km2 (500m possible?)
  - temporal freq: 7d
- domain: 
  - temporal: 2024- 
  - spatial: global
- detection threshold 100kg/h 


Technical docs and comparison w/ other CH4 satellite products 
[MethaneSAT Technical considerations-May 2019.pdf](https://github.com/user-attachments/files/16911880/MethaneSAT.Technical.considerations-May.2019.pdf)

**Reply by dlebauer** [reply]
At present, MethaneSAT data does not attempt to identify ag field sources

**Comment by dlebauer** [comment]
N2O flux data from Xia et al 2024 https://doi.org/10.1016/j.scitotenv.2024.171930

This is a summary of literature.
* key N2O variable is "Monthly N2O (units not given)  measured including units
* dataset includes >50 additional covariates for each record
* temporal + spatial resolution: site specific
* temporal + spatial domain: N america


Dataset (excel spreadsheet) Soil N₂O Emissions in Organic Farming Systems: A North American Meta-Dataset https://doi.org/10.7916/q0np-1w51

**Comment by infotroph** [comment]
Meta-analysis  of N2O emissions from specialty crops: Li et al 2024 https://onlinelibrary.wiley.com/doi/full/10.1111/gcb.17233

**Comment by dlebauer** [comment]
**Site Level delta SOC Data**

- [ ] (Russell Ranch) [Tautges et al 2019](https://doi.org/10.1111/gcb.14762) Deep soil inventories reveal that impacts of cover crops and compost on soil carbon sequestration differ in surface and subsurface soils https://github.com/ccmmf/organization/issues/237
- [ ] ~(Salinas Valley) [White et al 2020](https://doi.org/10.1016/j.dib.2020.106481) Soil carbon and nitrogen data during eight years of cover crop and compost treatments in organic vegetable production~  --> https://github.com/ccmmf/organization/issues/221

**Comment by dlebauer** [comment]
**N2O Global Datasets**

[Leschinsky and Xia,2025](https://doi.org/10.7916/q0np-1w51), Soil N₂O Emissions in Organic Farming Systems: A North American Meta-Dataset

[Lidong Li et al, 2024](https://datadryad.org/dataset/doi:10.5061/dryad.tht76hf5z). Soil nitrous oxide emissions from global specialty crop systems

Dorich, C.D., R.T. Conant, and D. Wang. 2021. Global N2O Database version 1.0 ver 1. Environmental Data Initiative. https://doi.org/10.6073/pasta/09a95cf44cd7835db002b0089ed5ffc3 (Accessed 2026-04-14).

**~Almond site N2O~**

~[Steenwerth 2025](https://catalog.data.gov/dataset/data-from-long-term-compost-use-and-high-frequency-low-concentration-fertigation-reduce-ns) Data from: Long-term compost use and high frequency low concentration fertigation reduce N<sub>2</sub>O emissions from a California almond orchard~ --> https://github.com/ccmmf/organization/issues/224

**Comment by dlebauer** [comment]
**Global SOC dataset**


**[only includes direction of effect; not magnitude]**
[Sanchez-Moreno et al 2025](https://www.nature.com/articles/s41597-025-05238-8) A global dataset of experimental agricultural management on soil carbon accrual, its synergies and trade-offs

**Comment by dlebauer** [comment]
From Modeling Framework Doc


| Source / Geography | Vegetation / Climate | Mgmt data avail? | N2O n sites/ siteyrs | CO2 n sites/ siteyrs | CH4 n sites/ siteyrs | SOC n sites/ siteyrs | Biomass n sites/ siteyrs |
| :---- | :---- | :---- | ----- | ----- | ----- | ----- | ----- |
| *databases* |  |  |  |  |  |  |  |
| **GRACEnet** | row crop \+ few grassland | complete | 29 | 27 | 26 | 21 | 6 |
| *US; mostly plains, Midwest, Southeast* | *any* |  | 115 | 106 | 99 | 157 | 26 |
| **Ameriflux\*** | cropland | refs† | 0 | 31 | 5 | refs† | TBD‡ |
| *Americas; mostly Western US* | *Mediterranean, semi-arid* |  | 0 | 119 | 39 | refs† | TBD‡ |
| **Ameriflux\*** | deciduous broadleaf | refs† | 0 | 8 | 0 | refs† | TBD‡ |
| *California and Mexico* | *semi-arid* |  | 0 | 41 | 0 | refs† | TBD‡ |
| **Ameriflux\*** | grassland | refs† | 0 | 14 | 3 | refs† | TBD‡ |
| *Western US* | *Mediterranean, semi-arid* |  | 0 | 133 | 16 | refs† | TBD‡ |
| **AgMIP** | row crops | complete | 0 | 0 | 0 | TBD‡ | 2182 |
| *Global* | *any* |  | 0 | 0 | 0 | TBD‡ | 2323 |
| **Li et al (2024)** | specialty crops | refs† | 114 | refs† | refs† | refs† | refs† |
| *Global* | *any* |  | TBD‡ | refs† | refs† | refs† | refs† |
| *primary reports* |  |  |  |  |  |  |  |
| **Tyler et al (2023)** | corn, alfalfa, pasture | refs† | 3 | 3 | 3 | 0 | 0 |
| *Sacramento Delta* | *Csa* |  | 11 | 11 | 11 | 0 | 0 |
| **Andrews et al (2021)** | alfalfa, sudan grass | complete (?) | 2 | 2 | 2 | 0 | 2 |
| *Imperial Valley* | *Csa* |  | 6 | 6 | 6 | 0 | 6 |
| **Nichols et al (2024)** | almond | partial | 1 | 1 | 1 | 0 | 1 |
| *Central Valley* | *Bsk* |  | 2 | 2 | 2 | 0 | 2 |
| **Nichols et al (2023)** | almond | partial | 1 | 1 | 1 | 1 | 0 |
| *Central Valley* | *Bsk* |  | 1 | 1 | 1 | 1 | 0 |
| **Jahanzad et al (2020, 2022\)** | almond | partial | 0 | 0 | 0 | 2 | 0 |
| *Central Valley* | *Bsk* |  | 0 | 0 | 0 | 2 | 0 |
| **Sánchez-Navarro et al (2022)** | almond | partial | 1 | 1 | 0 | 1 | 0 |
| *Spain* | *Bsk* |  | 3 | 3 | 0 | 2 | 0 |
| **Schellenberg et al (2012)** | almond | partial | 1 | 0 | 1 | 1 | 1 |
| *Kern Co* | *Bsk* |  | 2 | 0 | 2 | 1 | 2 |
| **Decock et al (2017)** | almond | complete(?) | 1 | 1 | 0 | 0 | 0 |
| *Colusa Co* | *Csa* |  | 3 | 3 | 0 | 0 | 0 |
| **Alsina et al (2013)** | almond | partial | 1 | 0 | 1 | 0 | 0 |
| *Colusa Co* | *Csa* |  | 1 | 0 | 1 | 0 | 0 |

Table 2: Potential validation data identified to date. Additional sites may be added.  
\*Counts for Ameriflux are filtered to sites with vegetation and climate comparable to CA. More sites may be available with a broader search.  
†This value is not compiled by the database, but may be available in source papers.  
‡This value is compiled by the database, but needs additional checks to determine which entries are usable for CCMMF.

**Comment by dlebauer** [comment]
@AritraDey-Dev FYI this discussion contains an inventory of potential validation data sources
